<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="I2E-LAP-RRA27" failures="2" tests="18" name="Testcase.QA_Meeting_Demo" time="102.887" errors="0" timestamp="2022-01-06T11:13:26 IST" skipped="0">
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_PageTitle" time="10.680"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_AfterEnterAValidCredentialTryToLoginLoginApp" time="5.819"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_WhenTwoClinicsArePresentThenCheckClinicPopupIsVisible" time="1.037"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_AndChooseSanjivaniClinicFromDropdownList" time="3.978"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_AllLeftNavigationMenuBarListArePresent" time="0.120"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_AppointmentPageisDisplayedByDefault" time="2.054"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_ColumnsNameInAppointmentPage" time="0.309">
    <failure message="expected:&lt;CLIENT&gt; but was:&lt;PATIENT&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<CLIENT> but was:<PATIENT>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at Testcase.QA_Meeting_Demo.Verify_ColumnsNameInAppointmentPage(QA_Meeting_Demo.java:401)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Verify_ColumnsNameInAppointmentPage -->
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_AddPatientPopupIsDisplayedAfterClickingAddPatientButton" time="4.254"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_FillTheNewPatientDetailsInTheAddPatientPopup" time="10.528"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_OpenOtherclass" time="10.163"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_CreateANewAppointment" time="12.809"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_PageTitleOfSettingPage" time="2.093"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_UpdateProfileSetting" time="2.002"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_AddClinicButtonIsPresentOnClinicSettingPage" time="4.246">
    <failure message="expected:&lt;Add Clinic&gt; but was:&lt;Add Clinic1&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<Add Clinic> but was:<Add Clinic1>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at Testcase.QA_Meeting_Demo.Verify_AddClinicButtonIsPresentOnClinicSettingPage(QA_Meeting_Demo.java:700)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Verify_AddClinicButtonIsPresentOnClinicSettingPage -->
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_CreateANewClinic" time="6.412"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_ContactUSPopupMessage" time="20.213"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_DoctorName" time="2.099"/>
  <system-out/>
  <testcase classname="Testcase.QA_Meeting_Demo" name="Verify_SignOutAppAfterClickingSignOutButton" time="4.071"/>
  <system-out/>
</testsuite> <!-- Testcase.QA_Meeting_Demo -->
